packages/eas-cli/src/build/ios/version.ts

```diff
@@ -10,34 +10,14 @@ import Log from '../../log';
 import { resolveWorkflowAsync } from '../../project/workflow';
 import { promptAsync } from '../../prompts';
 import { updateAppJsonConfigAsync } from '../utils/appJson';
-import { readPlistAsync, writePlistAsync } from './plist';
+import { readPlistAsync } from './plist';
 
 export enum BumpStrategy {
   SHORT_VERSION,
   BUILD_NUMBER,
   NOOP,
 }
 
-export async function bumpVersionAsync({
```

Comment:
This is no longer relevant since we are not allowing users to bump versions if they are not on the managed workflow.